symfony/symfony/CVE-2024-28859.yaml

```
@@ -0,0 +1,8 @@
+title:     Deserialization Gadget chain in Swift Mailer dependancy
+link:      https://github.com/FriendsOfSymfony1/symfony1/security/advisories/GHSA-wjv8-pxr6-5f4r
+cve:       CVE-2024-28859
+branches:
+    1.x:
+        time:     2024-02-27 20:26:56
+        versions: ['>=1.3.0', '<1.15.18']
+reference: composer://symfony/symfony
```

```suggestion
reference: composer://friendsofsymfony1/symfony1
```

(and the filename needs to changed accordingly)

---

@darkpills as symfony1 was never installable via composer, maybe it's correct to only have one file for symfony1 but keep the approach of two files for swiftmailer?